### DISCUSSION TITLE
How do we get grafana to use this date format: dd/mm/yyyy  hh: mm: ss (timestamp)

### DISCUSSION THREAD

**Discussion by lucattilia** [grafana/grafana#40160] [General]
**What i would like**
I want to use this date format in grafana: 05/10/2021 13:10:10

when i import the csv file into grafana, the format date is change :
![image](https://user-images.githubusercontent.com/92108560/136420206-fec1ddcd-f36e-4eda-a4a1-b4a165ebe8d5.png)

how i can fix this problem while using this format: dd/mm/yyyy hh:mm:ss

Thanks

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
